packages/app/src/shared/components/token-super-field/token-super-field.tsx

```diff
@@ -65,7 +66,7 @@ export function TokenSuperField({
       {Number(badgeValue) && badgeValue ? (
         <FieldContainer className={Suggestion}>
           <Label badge onClick={() => onChange(badgeValue)}>
-            Max {badgeValue}
+            Max {formatCompactCryptoCurrency(Number(badgeValue))}
```

Comment:
What about moving conversion to Number to `formatCompactCryptoCurrency`? We use it in a half of cases.

Reply:
Edited

Reply:
Great! Thank you.